dotCMS/src/integration-test/java/com/dotcms/rendering/velocity/viewtools/navigation/NavToolTest.java

```diff
@@ -204,8 +205,15 @@ public void testRootLevelNavigation_WhenOneFileAssetIsShownOnMenu() throws Excep
                     .getNav(site, systemFolder.getPath(), 1, user);
             assertNotNull(navResult);
 
+            /* method below `findShowOnMenuUnderFolder` expects a mutated version of SYSTEM_FOLDER with the hostId altered.
+               So let's create a defensive copy of the SYSTEM_FOLDER, modify it and pass it to the method.
+            */
+            Folder modifiedSystemFolder = new Folder();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'modifiedSystemFolder' could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5378894)